src/InsetOrderOptimizer.cpp

```diff
@@ -104,548 +80,94 @@ bool InsetOrderOptimizer::processInsetsIndexedOrdering()
     return added_something;
 }
 
-void InsetOrderOptimizer::moveInside()
+size_t InsetOrderOptimizer::getOuterRegionId(const VariableWidthPaths& toolpaths, size_t* p_number_of_regions)
```

Comment:
Because this one doesn't need to be managed (as in, deleted after use), since it's expected that it'll not be made with any 'new' (analogue). I guess I should make it into a reference instead (it's that stupid coding style I had to follow for years biting me again).

Reply:
Fixed.